.pytest_cache/v/cache/nodeids

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Why do we add this file? Can we delete it?

Reply:
Deleted